client/my-sites/woocommerce/woop/landing-page.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button } from '@automattic/components';
+import { useRef } from '@wordpress/element';
+import { translate } from 'i18n-calypso';
+import './style.scss';
+import Image01 from 'calypso/assets/images/woocommerce/woop-cta-image01.jpeg';
+import Image02 from 'calypso/assets/images/woocommerce/woop-cta-image02.jpeg';
+import Image03 from 'calypso/assets/images/woocommerce/woop-cta-image03.jpeg';
+import Image04 from 'calypso/assets/images/woocommerce/woop-cta-image04.jpeg';
+import CtaSection from 'calypso/components/cta-section';
+import FixedNavigationHeader from 'calypso/components/fixed-navigation-header';
+import MasonryWave from 'calypso/components/masonry-wave';
+import { addQueryArgs } from 'calypso/lib/url';
+
+interface Props {
+	startSetup: () => void;
+	isFeatureActive: boolean;
+	upgradingPlan: Record< string, unknown >;
+	siteSlug: string;
+}
+
+const images = [ { src: Image01 }, { src: Image02 }, { src: Image03 }, { src: Image04 } ];
+
+const WoopLandingPage: React.FunctionComponent< Props > = ( {
+	startSetup,
+	siteSlug,
+	isFeatureActive,
+	upgradingPlan,
+} ) => {
+	const navigationItems = [ { label: 'WooCommerce', href: `/woocommerce-installation` } ];
+	const ctaRef = useRef( null );
+
+	function onCTAHandler() {
+		if ( ! isFeatureActive && upgradingPlan?.product_slug ) {
+			return ( window.location.href = addQueryArgs(
+				{ redirect_to: window.location.href },
+				`/checkout/${ siteSlug }/${ upgradingPlan.product_slug }`
+			) );
+		}
+
+		startSetup();
+	}
+
+	return (
+		<div className="woop__landing-page">
+			<FixedNavigationHeader navigationItems={ navigationItems } contentRef={ ctaRef }>
+				<Button onClick={ onCTAHandler } primary>
+					{ isFeatureActive
+						? translate( 'Set up my store!' )
+						: translate( 'Upgrade to Premium plan' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Upgrade to a Premium Plan' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woop-landing-page-when-not-proper-plan)